glide.lock

```diff
@@ -1,8 +1,12 @@
-hash: b206ba15352a796ff142fc713255c9a56ca5bcef05c4951c698e7e8cf8ef2820
```

Comment:
i don't know anything about glide, do we really want a glide.lock file in git?  I'll assume it's fine. We can remove it later if need be.

Reply:
@jshaughn it has been discussed and maybe you already discussed this in chat too: that's common practice and it  proves any dependency update, since versions described in yaml may not always be strict versions. Same applies on the ui side with package lock file.

Reply:
Thanks for the info, @jotak.